device-connectors/src/testflinger_device_connectors/devices/__init__.py

```diff
@@ -195,6 +195,12 @@ def runtest(self, args):
         serial_port = config.get("serial_port")
         serial_proc = SerialLogger(serial_host, serial_port, "test-serial.log")
         serial_proc.start()
+
+        extra_env = {}
+        extra_env["agent_name"] = self.config.get("agent_name")
```

Comment:
```suggestion
        extra_env["AGENT_NAME"] = self.config.get("agent_name", "")
```

Just two things I would recommend here (proposed above):
1. make the env var all caps
2. make sure to get "" as a default value in case (unlikely) agent_name is missing, that way we avoid TypeError